2-farm/lessons/1-predict-plant-growth/translations/wio-terminal-temp.fr.md

```diff
@@ -0,0 +1,130 @@
+[![French](https://img.shields.io/badge/-French-purple)](wio-terminal-temp.fr.md)
```

Comment:
Can you remove the shields from these files please - we only need it on the main readme to direct readers to the appropriate translation from the root readme file.